src/gdrdrv/Makefile

```diff
@@ -45,7 +45,7 @@ REL_MINOR  := $(word 3,$(REL))
 
 all: gdrdrv.ko
 
-gdrdrv.ko:
+gdrdrv.ko: gdrdrv.c
```

Comment:
can we remove this one? it will be addressed in a separate PR

Reply:
Ok. Removed.